ekip/everykid/templates/base.html

```diff
@@ -56,21 +56,45 @@
 
     <section class="footer-content">
 
-      <h3>Every Kid In A Park</h3>
+      <p>Share your own pictures and #EveryKidInAPark story:</p>
+
+      <ul class="social">
+        <li>
+          <a href="http://www.instagram.com/everykidinapark" target="_blank">
+            <img src="{% static 'img/instagram.png' %}" alt="" />
+            <span>Instagram</span>
+          </a>
+        </li>
+        <li>
+          <a href="http://www.twitter.com/everykidinapark" target="_blank">
+            <img src="{% static 'img/twitter.png' %}" alt="" />
+            <span>Twitter</span>
+          </a>
+        </li>
+        <li>
+          <a href="http://www.facebook.com/everykidinapark" target="_blank">
+            <img src="{% static 'img/facebook.png' %}" alt="" />
+            <span>Facebook</span>
+          </a>
+        </li>
+      </ul>
 
       <nav>
         <ul>
           <li>
-            Get your pass
+            Become a partner school
```

Comment:
I think  we should remove the links here that we don't have content for yet. That way, we always have a working site.